ExtractLightsMerge

```diff
@@ -0,0 +1,392 @@
+/* -*-c++-*- */
+#pragma once
+
+#ifndef EXTRACT_LIGHTS_MERGE_H
+#define EXTRACT_LIGHTS_MERGE_H 1
+
+
+/**
+ * convert Env map Regions to Lights
+ */ 
+void createLightsFromRegions(const std::vector<satRegion>& regions, std::vector<light>& lights, const float *rgba, const uint width, const uint height, const uint nc)
```

Comment:
I **highly** disregards that exact kind of typedef, no gain whatsover, but lots of reading time wasted. 
Those typedef are always haf-assed, with no typedef for iterators, which is the real way to typedef template. 
reading "LightVector" you get to ask Is it a mathematical vector ? is it really a std::vector and not a std::list or std::deque because code author changed its mind and not the typedef name,  etc... 

A bag of bugs waiting to happen it is.
Always end with something hidden wrongly beneath typedef.
People _do gets burned_ using those typedef, _never_ when not using it, which is the paramount criteria of code practices.
That said, I can change it.

struct are lower case in envremap, higher case in cubemapcpp, seems I got confused.
(but that's old code already merged...)
Will Add Capital Letter